zebra-chain/src/sapling/spend.rs

```diff
@@ -134,6 +134,20 @@ impl ZcashSerialize for Spend<SharedAnchor> {
     }
```

Comment:
We should probably delete the `impl ZcashSerialize for Spend<SharedAnchor>`, because it's easy to use it and think you've serialised all the parts of the spend.